test/test_bin/test_check_mta_sts.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import boto3
+from botocore.exceptions import 
+from unittest.mock import patch
+
+import check_mta_sts
```

Comment:
I believe the interpreter will fall over here since this isn't a valid module path 🔥 

You'll need to update it to point to the file from the module, something along the lines of:

```suggestion
from bin import check_mta_sts
```